From 008669d7a7c128b4cf26c6b7fb78135a66c56a08 Mon Sep 17 00:00:00 2001
From: Alexander Koskovich <akoskovich@pm.me>
Date: Sat, 30 Dec 2023 18:28:11 -0500
Subject: [PATCH] lineage: overlay: Disable a couple more OTA upgrade
 components

Change-Id: I0ca22526897ef6539e55a2e270ea424a3e9a80ee
(cherry picked from commit bd4bdd0262fd6c817a0690d97b093ec7cdfe939c)
---
 overlay/common/lineage-sdk/lineage/res/res/values/config.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/overlay/common/lineage-sdk/lineage/res/res/values/config.xml b/overlay/common/lineage-sdk/lineage/res/res/values/config.xml
index 12509caaf9..7341671d5c 100644
--- a/overlay/common/lineage-sdk/lineage/res/res/values/config.xml
+++ b/overlay/common/lineage-sdk/lineage/res/res/values/config.xml
@@ -23,9 +23,11 @@
         <item>com.android.vending/com.google.android.finsky.systemupdateactivity.SettingsSecurityEntryPoint</item>
         <item>com.android.vending/com.google.android.finsky.systemupdateactivity.SystemUpdateActivity</item>
         <item>com.google.android.gms/com.google.android.gms.update.phone.PopupDialog</item>
+        <item>com.google.android.gms/com.google.android.gms.update.OtaSuggestionActivity</item>
         <item>com.google.android.gms/com.google.android.gms.update.OtaSuggestionSummaryProvider</item>
         <item>com.google.android.gms/com.google.android.gms.update.SystemUpdateActivity</item>
         <item>com.google.android.gms/com.google.android.gms.update.SystemUpdateGcmTaskService</item>
+        <item>com.google.android.gms/com.google.android.gms.update.SystemUpdatePersistentListenerService</item>
         <item>com.google.android.gms/com.google.android.gms.update.SystemUpdateService</item>
         <item>com.google.android.setupwizard/com.google.android.setupwizard.update.OtaUpdateActivity</item>
     </string-array>
